Makefile

```diff
@@ -15,134 +15,57 @@
 # Please note that this file was generated from [terraform-google-module-template](https://github.com/terraform-google-modules/terraform-google-module-template).
 # Please make sure to contribute relevant changes upstream!
 
-# Make will use bash instead of sh
-SHELL := /usr/bin/env bash
+SHELL := /usr/bin/env bash # Make will use bash instead of sh
 
-# Docker build config variables
-CREDENTIALS_PATH 			?= /cft/workdir/credentials.json
-DOCKER_ORG 				:= gcr.io/cloud-foundation-cicd
-DOCKER_TAG_BASE_KITCHEN_TERRAFORM 	?= 2.3.0
-DOCKER_REPO_BASE_KITCHEN_TERRAFORM 	:= ${DOCKER_ORG}/cft/kitchen-terraform:${DOCKER_TAG_BASE_KITCHEN_TERRAFORM}
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.0.1
+DOCKER_IMAGE_DEVELOPER_TOOLS := cft/developer-tools
+REGISTRY_URL := gcr.io/cloud-foundation-cicd
 
-# All is the first target in the file so it will get picked up when you just run 'make' on its own
-.PHONY: all
-all: check generate_docs
-
-# Run all available linters
-.PHONY: check
-check: check_shell check_python check_golang check_terraform check_docker check_base_files test_check_headers check_headers check_trailing_whitespace
-
-# The .PHONY directive tells make that this isn't a real target and so
-# the presence of a file named 'check_shell' won't cause this target to stop
-# working
-.PHONY: check_shell
-check_shell:
-	@source test/make.sh && check_shell
-
-.PHONY: check_python
-check_python:
-	@source test/make.sh && check_python
-
-.PHONY: check_golang
-check_golang:
-	@source test/make.sh && golang
-
-.PHONY: check_terraform
-check_terraform:
-	@source test/make.sh && check_terraform
-
-.PHONY: check_docker
-check_docker:
-	@source test/make.sh && docker
-
-.PHONY: check_base_files
-check_base_files:
-	@source test/make.sh && basefiles
-
-.PHONY: check_trailing_whitespace
-check_trailing_whitespace:
-	@source test/make.sh && check_trailing_whitespace
-
-.PHONY: test_check_headers
-test_check_headers:
-	@echo "Testing the validity of the header check"
-	@python test/test_verify_boilerplate.py
-
-.PHONY: check_headers
-check_headers:
-	@source test/make.sh && check_headers
-
-# Integration tests
-.PHONY: test_integration
-test_integration:
-	test/ci_integration.sh
-
-.PHONY: generate_docs
-generate_docs:
-	@source test/make.sh && generate_docs
-
-# Versioning
-.PHONY: version
-version:
-	@source helpers/version-repo.sh
-
-# Run docker
+ # The following tasks are provided as examples and aren't invoked from this
```

Comment:
Which targets is this comment referring to? My understanding is that all `docker_` targets are intended to be executed by developers on their hosts.

Reply:
You're right, removing.